src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -359,8 +359,30 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
   )
 
   banner(
-    s""" Usage: ${GoVerifier.name} -i <input-files> [OPTIONS] OR
-       |  ${GoVerifier.name} -p <directory> [OPTIONS]
+    s""" ${GoVerifier.name} supports three modes for specifying input files that should be verified:
+       |  ${GoVerifier.name} -i <input-files> [OPTIONS] OR
+       |  ${GoVerifier.name} -p <directories> <optional project root> [OPTIONS] OR
+       |  ${GoVerifier.name} -r <optional project root> <optional include and exclude package names> [OPTIONS]
+       |
+       | Mode 1 (-i):
+       |  The first way interprets the list of files as one package that should be verified.
+       |  Optionally, positional information can be provided for each file, e.g. <path to file>@42,111, such that only
+       |  members at these positions will be verified.
+       |
+       | Mode 2 (-p):
+       |  Interprets each provided directory as a directory storing files belonging to a single package.
```

Comment:
It seems ambiguous to me whether all files are treated as belonging to a single package overall vs a single package for each directory respectively.

Reply:
I can rephrase it to:
```suggestion
       |  Interprets each provided directory as a directory storing files belonging to a single package.
       |  ${GoVerifier.name} verifies all `.${PackageResolver.gobraExtension}` and `.${PackageResolver.goExtension} files` in the provided directories, while treating files in the same directory as belonging to the same package.
```